SF iOS/SF iOS/SceneDelegate.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SceneDelegate.swift
+//  TestingCreatingSwiftUIView
+//
+//  Created by Roderic Campbell on 6/8/19.
+//  Copyright © 2019 Roderic Campbell. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+    
+    var window: UIWindow?
+    
+    
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        
+        // Use a UIHostingController as window root view controller
+        let window = UIWindow(frame: UIScreen.main.bounds)
+        window.rootViewController = UIHostingController(rootView: FeedView(myEvents: eventData))
+        self.window = window
```

Comment:
@freddieoh  I think we'd have to do something like this in the group view controller when the user taps the cell. `UIHostingController` with our SwiftUI view as the root.